app/src/features/apiClient/screens/apiClient/components/sidebar/components/collectionsList/newRecordNameInput/NewRecordNameInput.tsx

```diff
@@ -58,7 +58,7 @@ export const NewRecordNameInput: React.FC<NewRecordNameInputProps> = ({ recordTo
 
       const wasForceRefreshed = await forceRefreshApiClientRecords();
       if (wasForceRefreshed && recordType === RQAPI.RecordType.COLLECTION) {
-        closeTabBySource(record.id, "collection", true);
+        closeTabBySource(record.id!, "collection", true);
```

Comment:
I think we wont need non null assertion here not. Correct me if thats not the case